tools/quic/quic_transport_server.py

```diff
@@ -16,8 +15,9 @@
 from aioquic.tls import SessionTicket
 
 SERVER_NAME = 'aioquic-transport'
+logger = logging.getLogger(__name__)
 
-handlers_path = None
+handlers_path = ""
```

Comment:
Single quotations are used in this file so doing so here is probably better for consistency.

Reply:
```suggestion
handlers_path = ''
```